keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -565,13 +566,13 @@ feature, can it break the existing applications?).
 
 NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
 -->
-Yes.
+Yes, we can just disable the feature gate.
 
 ###### What happens if we reenable the feature if it was previously rolled back?
 The policies are respected again.
 
 ###### Are there any tests for feature enablement/disablement?
-No, appropriate unit tests will be added for Alpha.
+Yes, both unit tests and integration tests are added.
```

Comment:
Can you please link them here?

Reply:
Updated @wojtek-t

Reply:
Those aren't typical enablement/disablement tests - those test the feature when it is enabled/disabled.
The enablement/disablement tests change whether the feature is enabled/disabled in the middle of the test.

On the scheduler side this seems to be good enough, because in scheduler this is basically "in-memory" feature.
So maybe please add something like:

"In the scheduler, this is in-memory feature, so only tests checking both feature being enabled and disabled were added".

However, this KEP is also introducing an API change, so a test similar to this one would be useful on the registry side:
https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282

Would you be able to add something like that?
I don't want to block this PR, so if you could add here that a strategy test will be added and add that to beta graduation criteria, I would be fine.

Reply:
I add the test here https://github.com/kubernetes/kubernetes/pull/112805, PTAL @wojtek-t

Reply:
@kerthcet - I don't want to block Beta on it - can you please add here that such test will be added (and add that to beta graduation criteria) and I will approve the PRR then?

Reply:
Updated the enablement tests again. PTAL.

Reply:
@kerthcet the KEP freeze is tomorrow. Just update this PR to say that the tests will be added (linking to the PR is fine too).